src/Microsoft.VisualStudio.Editor.Razor/ImportChangedEventArgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal class ImportChangedEventArgs : EventArgs
+    {
+        public ImportChangedEventArgs(string importPath, ImportChangeKind kind, IEnumerable<string> associatedDocuments)
+        {
+            ImportPath = importPath;
+            Kind = kind;
+            AssociatedDocuments = associatedDocuments;
+        }
+
+        public string ImportPath { get; }
```

Comment:
Import is inferred by the class name here. Call it `FilePath` to stay consistent with other types we own (like RazorSourceDocument).